_overviews/scala3-book/methods-most.md

```diff
@@ -359,30 +624,46 @@ Their main purpose is to let you add new functionality to closed classes.
 As shown in that section, imagine that you have a `Circle` class, but you can’t change its source code.
 For instance, it may be defined like this in a third-party library:
 
+{% tabs method_23 %}
+{% tab 'Scala 2 and 3' for=method_23 %}
+
 ```scala
 case class Circle(x: Double, y: Double, radius: Double)
 ```
 
+{% endtab %}
+{% endtabs %}
+
 When you want to add methods to this class, you can define them as extension methods, like this:
 
+{% tabs method_24 %}
+{% tab 'Scala 3 only' for=method_24 %}
+
 ```scala
 extension (c: Circle)
   def circumference: Double = c.radius * math.Pi * 2
   def diameter: Double = c.radius * 2
   def area: Double = math.Pi * c.radius * c.radius
 ```
 
+{% endtab %}
+{% endtabs %}
+
 Now when you have a `Circle` instance named `aCircle`, you can call those methods like this:
 
+{% tabs method_25 %}
+{% tab 'Scala 2 and 3' for=method_25 %}
```

Comment:
this does not make any sense to have a scala 2 and 3 tab when these methods were never defined in scala 2

Reply:
changed to Scala 3 Only.
I cannot write scala 2 code.